src/services/token.service.js

```diff
@@ -0,0 +1,33 @@
+import { Token } from '../models/index.js';
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({
+    where: {
+      userId,
+    },
+  });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
```

Comment:
Ensure that the `Token.create` method correctly handles the creation of a new token entry in the database. Verify that the `userId` and `refreshToken` fields are correctly defined in the `Token` model.